delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/event/GlobalEventStoreSuite.scala

```diff
@@ -90,4 +90,11 @@ class GlobalEventStoreSuite extends BioSuite with Doobie.Fixture with Doobie.Ass
     store.events(Offset.at(2L)).assert(envelope3, envelope4)
   }
 
+  test(s"Delete events for $id") {
+    for {
+      _ <- store.delete(id).transact(xas.write)
```

Comment:
This has got me thinking about writes in other places, and how they might occur in different transactions. I wonder if we have things which should (or even just could) be transacted together but are not

Reply:
This is something to have in mind every time we wrote something in db, especially when we are update an business entity so as to keep the event log and the computed state in sink for example